bochs/config.cc

```diff
@@ -3155,14 +3156,15 @@ static int parse_line_formatted(const char *context, int num_params, char *param
     }
     if (params[1][8] == '0') {
       BX_INFO(("Ignoring magic break points"));
-      bx_dbg.magic_break_enabled = 0;
     }
-    else if (params[1][8] == '1') {
-      BX_INFO(("Stopping on magic break points"));
-      bx_dbg.magic_break_enabled = 1;
+    else if (params[1][8] == '1'){
+      {
```

Comment:
you have plenty of example of Bochs style indentation around, please try to obbey the style